components/automate-ui/src/app/modules/policy/list/policy-list.component.html

```diff
@@ -30,9 +30,9 @@
                       <chef-table-cell class="status-column">{{ status(policy) | capitalize }}</chef-table-cell>
                       <chef-table-cell class="three-dot-column">
                       <!-- <app-authorized [allOf]="[`/iam/v2/policies/{id}`, 'delete', policy.id]"> -->
-                        <chef-control-menu *ngIf="policy.type !== 'CHEF_MANAGED'">
-                          <chef-option (click)="startPolicyDelete(policy)">Delete Policy</chef-option>
-                        </chef-control-menu>
+                        <mat-select panelClass="chef-control-menu-x" *ngIf="policy.type !== 'CHEF_MANAGED'">
```

Comment:
at the time it was about differentiating while building.  For the time being, i'll push through as `chef-control-menu-x` so that we don't delay, but i'll add to the list of cleanup items to be addressed.

Reply:
actually - quick search and replace should handle this now.